package.json

```diff
@@ -59,8 +59,10 @@
     "@sentry/integrations": "7.114.0",
     "@sentry/react-native": "5.33.1",
     "base-64": "^1.0.0",
+    "comlink": "^4.4.1",
     "cozy-client": "^52.0.0",
     "cozy-clisk": "^0.38.1",
+    "cozy-dataproxy-lib": "^2.3.0",
```

Comment:
It should be 2.4.0 I assume? To get https://github.com/cozy/cozy-libs/pull/2697

Reply:
It is not needed, 2.4.0 only impacts the DataProxyProvider that is hosted in web cozy-apps.

Reply:
Right, my bad